File: Hello_World_Tutorials/testing/python/mllist.py
from Bridges import *
from MLelement import *

class MLtutorial:
    bridges = Bridges(7, "YOUR_USER_ID", "YOUR_API_KEY")

    el0 = MLelement("0", "0")
    el1 = MLelement("1", "1")
    el2 = MLelement("2", "2")
    el3 = MLelement("3", "3")
    el4 = MLelement("4", "4")

    el0.set_next(el1)
    el1.set_next(el2)
    el2.set_next(el3)
    el3.set_next(el4)

    el5 = MLelement("5", "5")
    el6 = MLelement("6", "6")
    el7 = MLelement("7", "7")

    el5.set_next(el6)
    el6.set_next(el7)
    el2.set_sub_list(el5)

    el8 = MLelement("8", "8")
    el9 = MLelement("9", "9")

    el8.set_next(el9)
    el3.set_sub_list(el8)

    el10 = MLelement("10", "10")
    el11 = MLelement("11", "11")

    el10.set_next(el11)
    el6.set_sub_list(el10)

    el0.get_visualizer().set_size(40)
    el0.get_link_visualizer(el1).set_color("red")
    el0.get_link_visualizer(el1).set_thickness(3)
    el1.get_link_visualizer(el2).set_color("blue")
    el1.get_link_visualizer(el2).set_thickness(3)
    el2.get_link_visualizer(el3).set_color("cyan")
    el2.get_link_visualizer(el3).set_thickness(3)

    bridges.set_data_structure(el0)
    bridges.visualize()

File: Hello_World_Tutorials/testing/c++/mllist.cpp

#include "Bridges.h"
#include "MLelement.h"

using namespace bridges;

int main(int argc, char **argv)  {
	Bridges *bridges =  new Bridges(107, "YOUR_USER_ID", "YOUR_API_KEY");


	bridges->setTitle("A Trivial Multilist Example");

	//create  a linked list

	MLelement<int> *el0 = new MLelement<int>("0");
	MLelement<int> *el1 = new MLelement<int>("1");
	MLelement<int> *el2 = new MLelement<int>("2");
	MLelement<int> *el3 = new MLelement<int>("3");
	MLelement<int> *el4 = new MLelement<int>("4");

	el0->setNext(el1);
	el1->setNext(el2);
	el2->setNext(el3);
	el3->setNext(el4);
	// create a sublist from 2
	MLelement<int> *el5 = new MLelement<int>("5");
	MLelement<int> *el6 = new MLelement<int>("6");
	MLelement<int> *el7 = new MLelement<int>("7");

	el5->setNext(el6);
	el6->setNext(el7);

	el2->setSubList(el5);

	MLelement<int> *el8 = new MLelement<int>("8");
	MLelement<int> *el9 = new MLelement<int>("9");

	// create a second sublist from 3
	el8->setNext(el9);
	el3->setSubList(el8);

	// create a sublist from 5
	MLelement<int> *el10 = new MLelement<int>("10");
	MLelement<int> *el11 = new MLelement<int>("11");
	el10->setNext(el11);
	el6->setSubList(el10);

	el0->getVisualizer()->setSize (40);
	el0->getLinkVisualizer(el1)->setColor(Color("red"));
	el0->getLinkVisualizer(el1)->setThickness(3);
	el1->getLinkVisualizer(el2)->setColor(Color("blue"));
	el1->getLinkVisualizer(el2)->setThickness(3);
	el2->getLinkVisualizer(el3)->setColor(Color("cyan"));
	el2->getLinkVisualizer(el3)->setThickness(3);

	el0->getVisualizer()->setColor(Color("red"));
	//set visualizer type
	bridges->setDataStructure(el0);

	// visualize the list
	bridges->visualize();
}
